lib/raw_materials/interactors/rmt_bin_interactor.rb

```
@@ -260,7 +260,23 @@ def create_scanned_bin_groups(id, params) # rubocop:disable Metrics/AbcSize
         end
       end
 
-      create_bins(params, bin_asset_numbers)
+      res = create_bins(params, bin_asset_numbers)
+      unless res.failure?
+        bin_fullness = res.instance[0].bin_fullness
+        bins = res.instance.map { |b| { id: b.id, bin_asset_number: b.bin_asset_number } }
+        set_delivery_bin_samples(id, bins, bin_fullness)
+      end
+      res
+    rescue StandardError => e
+      failed_response(e.message)
```

Why are you rescuing std err here? It looks like an exception should not be swallowed here and only displayed to the user.